### PR TITLE
fix: play/pause player + hide player in chat views

### PR DESCRIPTION
**Pull request by JohnBerd**
play/pause player + hide player in chat views

### CODE REVIEW

File: js/packages/components/shared-components/StickyMusicPlayer.tsx [hunk 1]
```diff
@@ -6,15 +6,19 @@ import { useSafeAreaInsets } from 'react-native-safe-area-context'
 import { useStyles } from '@berty-tech/styles'
 import { useMusicPlayer } from '@berty-tech/music-player'
 import { useThemeColor } from '@berty-tech/store/hooks'
+import { navigationRef } from '@berty-tech/navigation'
 
 export const HEIGHT_OF_PLAYER = 60
 
+const hidePlayerRoutes: string[] = ['Chat.OneToOne', 'Chat.MultiMember', 'Chat.Group']
```
Comment (n0izn0iz) [inline js/packages/components/shared-components/StickyMusicPlayer.tsx:13]:
```suggestion
const hidePlayerRoutes: (keyof ScreenParams)[] = ['Chat.OneToOne', 'Chat.MultiMember', 'Chat.Group']
```
`ScreenParams` comes from `@berty-tech/navigation`

also maybe add
`export type ScreenNames = keyof ScreenParams` in `navigation/types`

Comment (JohnBerd) [reply js/packages/components/shared-components/StickyMusicPlayer.tsx:13]: Sure!

**Comment by zxxma** [comment]
LGTM 👍  (iOS review only)

**Review by n0izn0iz — APPROVED** [review]
LGTM

**Pull request merged**